### ISSUE TITLE
`Something wrong with Connection` when using --static 

### ISSUE THREAD

**Issue by xlight05** [ballerina-platform/ballerina-standard-library#4678]
**Description:**
$Subject.
This is only reproducible when we call external eps, uses --static flag with alpine.

Sample Code - 
```

import ballerina/http;
import ballerina/io;

public function main() returns error? {
    http:Client clientEP = check new("https://www.google.com/");
    http:Response res = check clientEP->/;
    io:println("Response status code: ", res.statusCode);
}
```
We need to see whats causing this issue.

**Suggested Labels (optional):**

**Suggested Assignees (optional):**

**Comment by xlight05** [comment]
Probably related to https://github.com/netty/netty/issues/12373 and https://netty.io/wiki/native-transports.html#using-the-linux-native-transport

**Comment by TharmiganK** [comment]
Yes, seems like a limitation of musl and netty. Hence removing the bug label.
Will follow up on this to give a resolution
For the moment, we recommend to use mostly-static images
